src/pages/status/migration/index.jsx

```diff
@@ -111,13 +112,20 @@ function Bar({ details }) {
     <>
       <h4>PRs made {details.progress.percentage.toFixed(0)}%</h4>
       <div className={styles.migration_details_bar}>
-        {ORDERED.filter(([key]) => details[key]?.length)
-          .map(([key], index) => (
-            <div
-              title={TITLES[key]}
-              className={styles[`${prefix}${key.replace("-", "_")}`]}
-              style={{ flex: details[key].length }} key={index}></div>
-          ))}
+        {ORDERED.filter(([key]) => details[key]?.length).map(([key], index) => (
+          <>
+          <a
+            id={"migration-bar-element" + "-" + key}
+            className={styles[`${prefix}${key.replace("-", "_")}`]}
+            style={{ flex: details[key].length }}
+            key={index}
+          ></a>
+         <Tooltip anchorSelect={"#migration-bar-element" + "-" + key}
+                  style={{color: "white", backgroundColor: "var(--ifm-color-gray-700)"}} place="top" className={styles.migration_details_bar_tooltip}>
```

Comment:
Where's this class defined? I think we can put it in `styles.modules.css` with the `style` attributes you have added?

Reply:
To be fair, I tried to use a class put in the `styles.modules.css` but it did not report the changes of colors so I decided to use the style attribute instead.

Reply:
I thought I removed the class attribute of the `Tooltip` component too but it is still there. If you know how to make it work with a class, I would be happy to know how to do it. Thanks

Reply:
Let me give it a try!

Reply:
Apparently we needed a more specific selector for this to work. See https://react-tooltip.com/docs/examples/styling#classes.

Reply:
> Apparently we needed a more specific selector for this to work. See https://react-tooltip.com/docs/examples/styling#classes.

I don't understand how it works with the `<style>` tag. I am having errors trying to translate the logics of the example in our specific case.